### ISSUE TITLE
Adding New Module: SHAPES & LINES

### ISSUE THREAD

**Issue by Azarasurti** [VaibhavCodeClub/learn#40]
Implementation of a module which would contain lessons regarding various shapes and lines.

**Comment by sapatevaibhav** [comment]
If you want to do it mention so...

**Comment by Vaibhav2154** [comment]
I find this Idea interesting. Please let me work on this Issue.

**Comment by Prashant-2024** [comment]
Can you please assign this issue to me under GSSOC'24.
I have experience with this type of Module addition in Flutter project.

**Comment by Vaibhav2154** [comment]
@Prashant-2024  Glad to hear that. I've made a PR with issue so you could list out improvisations once my PR is reviewed and merged, and let me know too so that even I get a good advise.
Thank You.

**Comment by Manikasethi** [comment]
Hey, Manika Sethi this side, I'm a GSSoC contributor and  I am confident that I can address this issue effectively since I have understood the code. Could you please assign it to me? Thanks!

**Comment by Azarasurti** [comment]
Yes, i want to implement this functionality. Kindly assign this to me.
